package.json

```diff
@@ -18,18 +18,20 @@
   "author": "Richard Rodger (http://richardrodger.com)",
   "license": "MIT",
   "dependencies": {
-    "fivebeans": "~1.1.1",
-    "nid": "~0.3.2",
-    "lodash": "~2.4.1"
+   "fivebeans": "~1.3.1",
+    "lodash": "^3.10.1",
+    "nid": "~0.3.2"
   },
   "devDependencies": {
-    "mocha": "~2.1.0",
+    "eslint-config-standard": "^4.4.0",
+    "eslint-plugin-standard": "^1.3.1",
+    "lab": "^6.2.0",
     "seneca": "plugin",
```

Comment:
You should use both dev and peer for seneca.


dev will install seneca for test purposes.
peer will ensure that seneca can be installed alongside beanstalk-transport